### PR TITLE
Rename universities to no longer use a prefix to be compatible with t…

### PR DESCRIPTION
**Pull request by verwoerd**
…he contest package format

Rename all image files. Update the organizations. Add contest package to the export script. Modify the automation logic to import the contest package using the `import-contest` script.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -187,4 +187,4 @@ For DOMJudge use the `background_64.zip`.
 |---|---|---|---|
 | <img src="./others/logos/O-1.svg" width="32" height="32"> | O-1 |  OBJECTION! | :heavy_check_mark: |
 | <img src="./others/logos/O-2.svg" width="32" height="32"> | O-2 |  The Wise Owl | :heavy_check_mark: |
-| <img src="./others/logos/O-3.svg" width="32" height="32"> | O-3 |  The Wise Owl | :heavy_check_mark: |
+| <img src="./others/logos/O-3.svg" width="32" height="32"> | O-3 |  Elektrichiens Vakbond | :heavy_check_mark: |
```
Comment (mpsijm) [inline README.md:190]: Maybe just this (also in https://github.com/WISVCH/icpc-logos/blob/contest-package/others/organizations.json#L15)? :stuck_out_tongue: 
```suggestion
| <img src="./others/logos/O-3.svg" width="32" height="32"> | O-3 | Electrotechnische Vereeniging | :heavy_check_mark: |
```

File: universities/organizations.json [hunk 2]
```diff
@@ -1,492 +1,492 @@
...
     "formal_name": "Universität Ulm",
     "country": "DEU"
   },
   {
-    "id": "U-2282",
+    "id": "2282",
     "name": "Leiden University",
     "formal_name": "Leiden University",
     "country": "NLD"
   },
```
Comment (mpsijm) [inline universities/organizations.json:139]: Possibly unrelated, but duplicate of `1088`

File: universities/organizations.json [hunk 3]
```diff
@@ -1,492 +1,492 @@
...
     "formal_name": "Tallinn University of Technology",
     "country": "EST"
   },
   {
-    "id": "U-11054",
+    "id": "11054",
     "name": "TalTech",
     "formal_name": "Tallinn University of Technology",
     "country": "EST"
   },
```
Comment (mpsijm) [inline universities/organizations.json:445]: Probably unrelated, but duplicate university (`11048` and `11054`)

File: .github/workflows/make_logos.yaml [hunk 4]
```diff
@@ -30,7 +30,7 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v2
```
Comment (mpsijm) [inline .github/workflows/make_logos.yaml:32]: Again, unrelated, but the [Actions](https://github.com/WISVCH/icpc-logos/actions/runs/11439948342) page shows warning that this version is deprecated. I believe the latest is `@v4`?

(same for other jobs, of course)

**Review by mpsijm — APPROVED** [review]
The changes and the artefacts generated by the GitHub Actions are looking good :smile: I only have some unrelated comments (see below):

File: universities/organizations.json (hunk 2 above)
Comment (verwoerd) [reply universities/organizations.json:139]: yes, this is one of the duplicates in ICPC

File: .github/workflows/make_logos.yaml (hunk 4 above)
Comment (verwoerd) [reply .github/workflows/make_logos.yaml]: hmm, I updated it, but the actions page still claims we use it. Did I miss any?

File: universities/organizations.json (hunk 3 above)
Comment (verwoerd) [reply universities/organizations.json]: One off them was wrong. After checking in ICPC only a single version exists. removed the wrong duplicate.

**Pull request merged**